--- prv.aff	2020-04-21 19:37:08.243878005 +0200
+++ new.aff	2020-04-21 19:37:08.239877920 +0200
@@ -1,7 +1,7 @@
 SET UTF-8
 TRY  -ernitsaoldgkupmchbvjwfzyx'’ëqïéèö.êü12635407çà8û9îñäôERNITSAOLDGKUPMCHBVJWFZYX
-KEY qwertyuiop|asdfghjkl|zxcvbnm|qawsedrftgyhujikolp|azsxdcfvgbhnjmk|aze|qsd|lm|wx|aqz|qws|
-WORDCHARS '’0123456789ĳ.-\/
+KEY qwertyuiop|asdfghjkl|zxcvbnm|qawsedrftgyhujikolp|azsxdcfvgbhnjmk|aze|qsd|lm|wx|aqz|qws|1234567890
+WORDCHARS '’0123456789ĳĲ.-\/
 NOSPLITSUGS
 MAXCPDSUGS 1
 ONLYMAXDIFF 
@@ -13,26 +13,35 @@
 FLAG long
 FORBIDDENWORD Fw
 NOSUGGEST NS
-MAP 5
-MAP eéèëê
-MAP aáà
-MAP iíìïî
-MAP oóòöô
-MAP uüúùû
-ICONV 9
-ICONV áá aa
-ICONV éé ee
-ICONV íé ie
-ICONV óó oo
-ICONV úú uu
-ICONV óé oe
+MAP 7	#TODO which mappings
+MAP eéèëêEÉÈËÊ	#TODO optimize characters and order
+MAP aáàäâåAÁÀÄÂÅ	#TODO optimize characters and order
+MAP iíìïîIÍÌÏÎ	#TODO optimize characters and order
+MAP oóòöôøOÓÒÖÔØ	#TODO optimize characters and order
+MAP uüúùûUÜÚÙÛ	#TODO optimize characters and order
+MAP cçCÇ	#TODO optimize characters and order https://nl.wikipedia.org/wiki/Cedille
+MAP nñNÑ	#TODO optimize characters and order https://nl.wikipedia.org/wiki/Tilde
+ICONV 16
+ICONV áá aa # dáár, dráái, tááie, ááien, beáám
+ICONV áó ao # Raoul TODO check
+ICONV áú au # láúw, áú
+ICONV éá ea # cadéáu
+ICONV éé ee # néé, lééuw
+ICONV éí ei # béíde héíig
+ICONV éú eu # kéúen
+ICONV íé ie # wíé, níéuw, officíéel
+ICONV óé oe # móét, snóéi, góéie
+ICONV óó oo # óók, óóit
+ICONV óú ou # góúd, Lóúis
+ICONV úí ui # geúít
+ICONV úú uu # natúúrlijk
 ICONV ’ '
 ICONV ij ĳ
 ICONV IJ Ĳ
 OCONV 2
 OCONV ĳ ij
 OCONV Ĳ IJ
-REP 593
+REP 706
 REP a's$ a	# centra's=>centra
 REP a's$ ums	# centra's=>centrums
 REP pt$ bd	# dichtgeslipt
@@ -127,8 +136,8 @@
 REP oi o-i
 REP oo o-o
 REP ou o-u
-REP ui u-i
-REP uu u-u	# tutuuitzet => tutu-uitzet
+REP ui u-i	# klinkerbotsing
+REP uu u-u	# klinkerbotsing
 REP te tte	# om verwarring met verleden tijd te corri
 REP s$ 's$	# scène's=>scènes
 REP s ss
@@ -618,50 +627,185 @@
 REP ei ie	# gebeid=>gebied
 REP heidstel stel	# zekerheidstelling=>zekerstelling
 REP ij ei	# Ij=>ei
-REP intelligentie inlichtingen	# intelligentiediensten_=_inlichtingendien
-REP roestvrijstaal roestvrij staal
+REP intelligentie inlichtingen	# intelligentiediensten_=_inlichtingendiensten
+REP ^roestvrijstaal$ roestvrij staal
 REP gebeid gebied
 REP dattie dat hij
 REP veras verras
 REP aanvaart aanvaardt
 REP persentage percentage
 REP deja-vu déjà_vu
+REP sjiek chic	# http://taaladvies.net/taal/advies/vraag/
+REP sjieke chique	# http://taaladvies.net/taal/advies/vraag/
+REP sjieker chiquer	# http://taaladvies.net/taal/advies/vraag/
+REP sjiekst chicst	# http://taaladvies.net/taal/advies/vraag/
+REP asseccoires accessoires
+REP aseccoires accessoires
+REP assecoires accessoires
+REP asecoires accessoires
+REP asseccoire accessoire
+REP aseccoire accessoire
+REP assecoire accessoire
+REP asecoire accessoire
+REP asseccoires accessoir
+REP aseccoir accessoir
+REP assecoir accessoir
+REP asecoir accessoir
+REP ^voorschut$ voor_schut
+REP ^M m	# onderkast_ook_suggereren
+REP ^A a
+REP ^B b	# ook onderkast suggereren
+REP ^C c	# ook onderkast suggereren
+REP ^D d	# ook onderkast suggereren
+REP ^E e	# ook onderkast suggereren
+REP ^F f	# ook onderkast suggereren
+REP ^G g	# ook onderkast suggereren
+REP ^H h	# ook onderkast suggereren
+REP ^I i	# ook onderkast suggereren
+REP ^J j	# ook onderkast suggereren
+REP ^K k	# ook onderkast suggereren
+REP ^L l	# ook onderkast suggereren
+REP ^N n	# ook onderkast suggereren
+REP ^O o	# ook onderkast suggereren
+REP ^P p	# ook onderkast suggereren
+REP ^Q q	# ook onderkast suggereren
+REP ^R r	# ook onderkast suggereren
+REP ^S s	# ook onderkast suggereren
+REP ^T t	# ook onderkast suggereren
+REP ^U u	# ook onderkast suggereren
+REP ^V v	# ook onderkast suggereren
+REP ^W w	# ook onderkast suggereren
+REP ^X x	# ook onderkast suggereren
+REP ^Y y	# ook onderkast suggereren
+REP ^Z z	# ook onderkast suggereren
+REP euren$ eurs
+REP ^het het_	# hetwoord,_spatie_suggereren
+REP ^dat dat_
+REP ^de de_
+REP prijsstel prijs_stel
+REP atie$ ering
+REP ering$ atie
+REP orches orkes	# orkestratie
+REP ^oet moet
+REP ^dete$ de_te
+REP murf murw
+REP tje 'tje
+REP servie servië
+REP ^da's$ dat is	# om_suggestie_vooraan_te_krijgen
+REP pvda PvdA	# om_suggestie_vooraan_te_krijgen
+REP ^bvb$ bijv.
+REP ^eeuwse$ -eeuwse
+REP tengevolge ten gevolge	# om_suggestie_vooraan_te_krijgen
+REP event evenement
+REP vs vs.
+REP apps applicaties
+REP sch s	# ouderwets naar modern
+REP ^fun$ lol
+REP aie aïe	# voor_naar_voren_halen_suggestie_o.a._naïef
+REP openlucht open lucht
+REP ^kinder$ kinder-
+REP events evenementen	# niet per se fout, maar lelijk
+REP ^danpas$ dan pas	# spatie suggereren
+REP email e-mail	# om_suggestie_vooraan_te_krijgen
+REP effe even	# om_suggestie_vooraan_te_krijgen
+REP ^ne$ een	# Vlaams_naar_Nederlands
+REP ^zeide$ zei	# ouderwets,_Bijbels
+REP -tje 'tje	# cd-tje_=>_cd'tje
+REP ie ĳ	# biezonder_=>_bijzonder
+REP mss misschien	# forum-taal
+REP nen een	# Vlaams_naar_Nederlands
+REP ect etc	# et_cetera
+REP ^'m$ hem
+REP ^mn$ mijn
+REP ^'n$ een
+REP -ers$ 'ers	# hob-ers_=>_hbo'ers
+REP kkies kjes	# strakkies_=>_strakjes
+REP presies precies	# om_suggestie_vooraan_te_krijgen
+REP ^zonder zonder_	# spatie suggereren
+REP -e ë	# ge-ent_=>_geënt
+REP kkie kje	# bakkie_=>_bakje
+REP ppie pje	# chippie_=>_chipje
+REP g ge	# aangstuurd=>aangestuurd
+REP ees ace	# rees=>race
+REP $cariere^ carrière	# suggestie was niet aanwezig
+REP zijn$ zijn_	# spatie suggereren
+REP van$ _van	# spatie suggereren
+REP wordt$ _wordt	# spatie suggereren
+REP ^wordt wordt_	# spatie suggereren
+REP bruid bruids	# anders_komt_bruis_als_suggestie
+REP vvd VVD	# anders_geen_suggestie
+REP ship schap	# partnership=>partnerschap
+REP gske kje	# verbindingske=>verbindinkje
+REP zgn zgn.
+REP tten s	# schoolboycotten=>boycots
+REP terzelfdertijd terzelfder_tijd
+REP ingskes inkjes
+REP ingskes ingetjes
+REP bijbel Bijbel	# om_goede_suggestie_te_krijgen
+REP derin ster	# bewaardering_=>_bewaarster
+REP nerin onster	# bewonerin=>bewoonster
+REP nerinnetje onstertje	# bewonerinnetje=>bewoonstertje
+REP gemeubuleerde gemeubileerde	# suggestie was niet aanwezig
+REP volklied volkslied	# volklied=>volkslied
+REP ^zer$ zeer	# zer=>zeer
 COMPOUNDMIN 0
-COMPOUNDRULE 37
-COMPOUNDRULE (N1)(n2)	  	# eenen+zestig[ste]
-COMPOUNDRULE (Nh)(n3)		# honderden+twee[de]
-COMPOUNDRULE (Nh)(n2)		# honderden+twintig[ste]
-COMPOUNDRULE (Nh)(N1)(n2)	# honderden+eenen+dertig
-COMPOUNDRULE (N4)(NH)		# vijf+honderd 
-COMPOUNDRULE (N4)(Nh)(n3)	# vijf+honderden+een 
-COMPOUNDRULE (N4)(Nh)(n2)	# vijf+honderd+twintig 
-COMPOUNDRULE (N4)(Nh)(N1)(n2)	# vijf+honderd+eenen+zestig
-COMPOUNDRULE (N4)(Nd)		# vijf+duizend
-COMPOUNDRULE (N2)(Nd)		# zestig+duizend[ste]
-COMPOUNDRULE (N1)(N2)(Nh)	# eenen+zestig+honderd
-COMPOUNDRULE (N1)(N2)(Nh)(n3)	
-COMPOUNDRULE (N1)(N2)(Nh)(n2)
-COMPOUNDRULE (N1)(N2)(Nh)(N1)(n2)
-COMPOUNDRULE (N1)(N2)(Nd)
-COMPOUNDRULE (NH)(Nd)
-COMPOUNDRULE (NH)(N3)(Nd)
-COMPOUNDRULE (NH)(N2)(Nd)
-COMPOUNDRULE (N1)(N2)(Nh)(N3)(Nd)
-COMPOUNDRULE (N1)(N2)(Nh)(N2)(Nd)
-COMPOUNDRULE (N1)(N2)(Nh)(N1)(N2)(Nd)
-COMPOUNDRULE (NH)(N1)(N2)(Nd)
-COMPOUNDRULE (N4)(NH)(Nd)
-COMPOUNDRULE (N4)(Nh)(N3)(Nd)
-COMPOUNDRULE (N4)(Nh)(N2)(Nd)
-COMPOUNDRULE (N4)(Nh)(N1)(N2)(Nd)
-COMPOUNDRULE (N1)(N2)(NH)(Nd)	# eenentwintighonderdduizend
-COMPOUNDRULE (N1)(N2)(Nj)   	# eenen+zestig+jarig
-COMPOUNDRULE (N3)(Nj)   	# acht+jarig
-COMPOUNDRULE (N2)(Nj)   	# twintig+jarig
-COMPOUNDRULE (N1)(N2)(Nj) 	# eenen+zestig+jarig
-COMPOUNDRULE (G1)(G0)(NJ)  	# 12-jarig~
-COMPOUNDRULE (G1)(G0)(G0)(NJ)  	# 102-jarig~
-COMPOUNDRULE (G1)(NJ)  		# 8-jarig~
+COMPOUNDRULE 48
+COMPOUNDRULE 40
+COMPOUNDRULE (Nn)(NT)							# eenen twintig[ste]
+COMPOUNDRULE (Nh)(NG)							# honderd vijftien[de]
+COMPOUNDRULE (Nh)(NT)							# honderd twintig[ste]
+COMPOUNDRULE (Nh)(Nn)(NT)						# honderd eenen twintig[ste]
+COMPOUNDRULE (Na)(NH)							# vijf honderd[ste]
+COMPOUNDRULE (Na)(Nh)(NG)						# vijf honderd vijftien[de]
+COMPOUNDRULE (Na)(Nh)(NT)						# vijf honderd twintig[ste]
+COMPOUNDRULE (Na)(Nh)(Nn)(NT)					# vijf honderd eenen twintig[ste]
+COMPOUNDRULE (Nn)(Nt)(NH)						# eenen twintig honderd[ste]
+COMPOUNDRULE (Nn)(Nt)(Nh)(NG)					# eenen twintig honderd vijftien[de]
+COMPOUNDRULE (Nn)(Nt)(Nh)(NT)					# eenen twintig honderd twintig[ste]
+COMPOUNDRULE (Nn)(Nt)(Nh)(Nn)(NT)			# eenen twintig honderd eenen twintig[ste]
+COMPOUNDRULE (Ng)(Nd)							# tien duizend[ste]
+COMPOUNDRULE (Nt)(Nd)							# twintig duizend[ste]
+COMPOUNDRULE (Nn)(Nt)(Nd)						# eenen twintig duizend[ste]
+COMPOUNDRULE (Nh)(Ng)(Nd)						# honderd tien duizend[ste]
+COMPOUNDRULE (Nh)(Nt)(Nd)						# honderd twintig duizend[ste]
+COMPOUNDRULE (Nh)(Nn)(Nt)(Nd)					# honderd eenen twintig duizend[ste]
+COMPOUNDRULE (Na)(Nh)(Ng)(Nd)					# vijf honderd tien duizend[ste]
+COMPOUNDRULE (Na)(Nh)(Ng)(Nd)(NG)			# vijf honderd tien duizend vijftien[de]
+COMPOUNDRULE (Na)(Nh)(Ng)(Nd)(NT)			# vijf honderd tien duizend twintig[ste]
+COMPOUNDRULE (Na)(Nh)(Ng)(Nd)(Nn)(NT)		# vijf honderd tien duizend eenen twintig[ste[
+COMPOUNDRULE (Na)(Nh)(Nt)(Nd)					# vijf honderd twintig duizend[ste]
+COMPOUNDRULE (Na)(Nh)(Nt)(Nd)(NG)			# vijf honderd twintig duizend vijftien[de]
+COMPOUNDRULE (Na)(Nh)(Nt)(Nd)(NT)			# vijf honderd twintig duizend twintig[ste]
+COMPOUNDRULE (Na)(Nh)(Nt)(Nd)(Nn)(NT)		# vijf honderd twintig duizend eenen twintig[ste]
+COMPOUNDRULE (Na)(Nh)(Nn)(Nt)(Nd)			# vijf honderd eenen twintig duizend[ste]
+COMPOUNDRULE (Nn)(Nt)(Nh)(Ng)(Nd)			# eenen twintig honderd tien duizend[ste]
+COMPOUNDRULE (Nn)(Nt)(Nh)(Nt)(Nd)			# eenen twintig honderd twintig duizend[ste]
+COMPOUNDRULE (Nn)(Nt)(Nh)(Nn)(Nt)(Nd)		# eenen twintig honderd eenen twintig duizend[ste]
+COMPOUNDRULE (Ng)(Nj)							# twaalf jarig[e]
+COMPOUNDRULE (Nt)(Nj)							# twintig jarig[e]
+COMPOUNDRULE (Nn)(Nt)(Nj)						# eenen twintig jarig[e]
+COMPOUNDRULE (Nh)(Ng)(Nj)						# honderd vijftien jarig[e]
+COMPOUNDRULE (N*)*(NJ)							# 112-jarig
+COMPOUNDRULE (N*)*(N1)(N*)(Ld)				# ..315 de
+COMPOUNDRULE (N*)*(N2)(N*)(Ls)				# ..321 ste
+COMPOUNDRULE (N*)*(N0)(N0)(Ls)				# ..300 ste
+COMPOUNDRULE (N*)*(N0)(N1)(Ls)				# ..301 ste
+COMPOUNDRULE (N*)*(N0)(N2)(Ld)				# ..302 de
+COMPOUNDRULE (Rt)(Re)
+COMPOUNDRULE (Rc)(Re)
+COMPOUNDRULE (Rc)(Rt)
+COMPOUNDRULE (Rc)(Rt)(Re)
+COMPOUNDRULE (Rm)(Re)
+COMPOUNDRULE (Rm)(Rt)
+COMPOUNDRULE (Rm)(Rt)(Re)
+COMPOUNDRULE (Rm)(Rc)
+COMPOUNDRULE (Rm)(Rc)(Re)
+COMPOUNDRULE (Rm)(Rc)(Rt)
+COMPOUNDRULE (Rm)(Rc)(Rt)(Re)
+COMPOUNDRULE (rt)(re)
+COMPOUNDRULE (rc)(re)
+COMPOUNDRULE (rc)(rt)
+COMPOUNDRULE (rc)(rt)(re)
 COMPOUNDRULE (Sv)(Sn)		# -straat, -plein etc.
 COMPOUNDRULE (Ll)(Hp)(Lr)	# Nederlands-Ghanese
 COMPOUNDRULE (Ll)(Hp)(Ll)(Le)	# Nederlands-Duits+e
